spec/requests/api/editor/editor_can_put_articles_to_archive_spec.rb

```diff
@@ -40,15 +44,17 @@
     let(:journalist_headers) { journalist.create_new_auth_token }
 
     before do
-      delete "/api/articles/#{article.id}", headers: journalist_headers
+      put "/api/articles/#{article.id}",
+          params: { status: 'archived' },
+          headers: journalist_headers
     end
 
     it 'is expected to return status code 403' do
       expect(response).to have_http_status 403
     end
 
     it 'is expected to return an error message' do
-      expect(response_json['error_message']).to eq 'You are not authorized to delete this article'
+      expect(response_json['error_message']).to eq 'You are not authorized to edit this article'
     end
```

Comment:
I'm not sure we're using these messages on the front end anyway, but the journalist might be able to edit it - just not change the status :P

Reply:
yes, but it requires another condition to check what we are doing, we deicided that it is not worth it. Refactoring chore?